LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Views/TunnelControllerView/TunnelControllerViewModel.swift

```diff
@@ -140,29 +140,15 @@ public final class TunnelControllerViewModel: ObservableObject {
 
     private func subscribeToStatusChanges() {
         statusReporter.statusObserver.publisher
-            .subscribe(on: Self.statusDispatchQueue)
-            .sink { [weak self] status in
-
-            guard let self else {
-                return
-            }
-
-            Task { @MainActor in
-                self.connectionStatus = status
-                switch status {
-                case .connected, .connecting:
-                    self.isVPNEnabled = true
-                default:
-                    self.isVPNEnabled = false
-                }
-            }
```

Comment:
This was duplicated in the code I simplified above... so it can now be reduced and simplified.